FirebaseAuth/Sources/SystemService/FIRAuthStoredUserManager.m

```diff
@@ -81,6 +81,9 @@ - (FIRUser *)getStoredUserForAccessGroup:(NSString *)accessGroup
   query[(__bridge id)kSecAttrAccessGroup] = accessGroup;
   query[(__bridge id)kSecAttrService] = projectIdentifier;
   query[(__bridge id)kSecAttrAccount] = kSharedKeychainAccountValue;
+  if (@available(iOS 13.0, macOS 10.15, macCatalyst 13.0, tvOS 13.0, watchOS 6.0, *)) {
+    query[(__bridge id)kSecUseDataProtectionKeychain] = (__bridge id)kCFBooleanTrue;
+  }
```

Comment:
So IIUC: when the `kSecUseDataProtectionKeychain` is added to the query, the prompt on macOS is prevented. When it's not (i.e. we are on running on pre-macOS 10.15), the prompt will appear?

Reply:
@ncooke3 yes. If macOS pre 10.15 don't behave differently, then on that system, after reinstall/update or when the account is checked from an extension there will be a keychain access prompt.

Reply:
For <10.15 a working sulution may be to set `shareAuthStateAcrossDevices` to true. because [AccessGroup documentation](https://developer.apple.com/documentation/security/ksecattraccessgroup) says 

> This attribute applies to macOS keychain items only if you also set a value of true for the kSecUseDataProtectionKeychain key, the kSecAttrSynchronizable key, or both.